wiki/Contests/FBC/fr.md

```diff
@@ -0,0 +1,160 @@
+---
+tags:
+- 'French Beatmapping Contest'
+- 'FBC'
+- 'Mapping' 
+---
+
+# French Beatmapping Contest
+
+Le **French Beatmapping Contest** (***FBC***) est un tournoi de mapping uniquement accessible aux personnes francophones. Dans chaque édition, les mappeurs doivent réaliser une difficulté de niveau Insane ou plus. 
+
+Les points sont attribués en suivant les critères de notation suivants: **Structure**, **Pertinence**, **Créativité**, **Impression du juge** et **Hitsounds**. Il est possible que les critères changent si nécessaire.  
+
+Les trois premiers mappeurs d'un tournoi recevront chacun un titre de osu!supporter d'une durée qui peut varier en fonction des éditions. Cependant, le vainqueur de chaque édition recevra un badge personalisé qui s'affichera sur son profil osu!.
+
+Depuis l'édition 2017, les musiques choisies ont une durée inférieure à 3 minutes et les difficultés en dessous du niveau Insane sont mappés par les organisateurs des tournois ou bien par des membres de la communauté de mapping francophone. 
+
+# FBC 2015
```

Comment:
Mauvaise nouvelle, il ne doit y avoir qu'un seul header de niveau 1, donc tous les autres headers suivants doivent être passés au niveau suivant.

Il y a le même problème dans l'article anglais.

```suggestion
## FBC 2015
```

Reply:
@Joehuu do you have any opinion on this? from what I can tell we don't usually put level 1 headers within articles, but it doesn't really break anything and seems appropriate enough to me

Reply:
https://osu.ppy.sh/help/wiki/Skinning/Sounds#countdown

![](https://puu.sh/E4sGc/3bb3f55023.png)

Yeah only one level one header, being the article title according to ASC. Visually, the TOC will have this big spacing which shouldn't happen.

Reply:
Alright, I'll fix then! Thanks for replying!